.goreleaser.yaml

```diff
@@ -0,0 +1,44 @@
+project_name: kubectl-open-notebook
+before:
+  hooks:
+    - go mod tidy
+    - go generate ./...
+builds:
+  - main: ./kubectl/open-notebook/
+    env:
+      - CGO_ENABLED=0
+    goos:
+      - linux
+      - windows
+      - darwin
```

Comment:
Do Mac and Windows work? Did you test those? I've only been testing on Linux so far

Reply:
Mac works and I've tested Linux end to end. Let me give it a quick test on Windows to see how it goes.

The `bash -c "curl...` will definitely fail on Windows if it's not WSL. Let me verify Windows on WSL x86_64

Reply:
works here:

```
Linux raisin 5.10.16.3-microsoft-standard-WSL2 #1 SMP Fri Apr 2 22:23:49 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux
```

Reply:
Manually downloading the Windows binary worked on W11:

```
PS C:\Users\brand\Downloads\kubectl-open-notebook_Windows_x86_64> .\kubectl-open-notebook.exe
2023/07/10 23:00:14 must provide filename flag (-f) or notebook name
```